cluster-provision/gocli/hack/install-go.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -xe
+
+destination=$1
```

Comment:
Why do we need this script?

Reply:
Is a simplified version of go gimme from travis, it install go under `build` directory so we vendor the go compiler.

Reply:
Why do we not use an image where go is installed?

Reply:
The idea is to post gocli binaries at release page at github no gocli container will exists, so we do a plain compilation with a vendored go.